code/lib/cli/src/automigrate/fixes/sb-scripts.ts

```diff
@@ -95,13 +122,14 @@ export const sbScripts: Fix<SbScriptsRunOptions> = {
       )} respectively.`,
       `In order to work with ${sbFormatted}, Storybook's ${chalk.magenta(
         'storybook'
-      )} binary has to be installed and your storybook scripts have to be adjusted to use the binary. We can install the storybook binary and attempt to adjust your scripts for you.`,
+      )} binary has to be installed and your storybook scripts have to be adjusted to use the binary. We can install the storybook binary and adjust your scripts for you:\n`,
+      newScriptsMessage.join('\n\n'),
     ].join('\n');
 
     return [
```

Comment:
Can we just do a `dedent\` block that's readable instead of this weird join business?

Reply:
if I remember correctly I used the weird join business because dedent won't work if you're not putting everything in the same dedent block. I'll try it though!

Reply:
changes made!